interface/interface.dm

```diff
@@ -66,14 +66,14 @@
 	if(restricted_mode || is_banned_from(ckey, "Bug Report"))
 		to_chat(src, span_warning("You are not currently allowed to make a bug report through this system."))
 		return
-	var/message = "This will start reporting an issue, gathering some information from the server and your client, before submitting it to github."
-	if(GLOB.revdata.testmerge.len)
-		message += "<br>The following experimental changes are active and may be the cause of any new or sudden issues:<br>"
-		message += GLOB.revdata.GetTestMergeInfo(FALSE)
+	if(alert(src, "This will start reporting an issue, gathering some information from the server and your client, before submitting it to github.", "Report Issue","Continue","Abort")!="Continue")
```

Comment:
tgui alert. it will turn into a normal alert for people who have tgui alerts off